INSTALL

```diff
@@ -24,14 +25,10 @@ Table of contents:
 
 The following packages must be installed to use Beluga:
 
-ocaml 3.12.2+:           http://caml.inria.fr/download.en.html
+ocaml 4.07.1+:           http://caml.inria.fr/download.en.html
```

Comment:
One of dependencies of one of the dependencies is a library called "dune" which wouldn't compile with 3.12.2 nor 4.02.0, but did compile with 4.07.1; I can try to find a more accurate minimum version if you prefer.